portfolio/src/main/webapp/index.html

```diff
@@ -59,6 +59,14 @@ <h4>
       <button type="button" class="btn btn-info" onclick="getRandomFact()">random fact</button>
       <div id="fact-container"></div>
     </div>
+    <div class="comment-section">
+      <form action="/data" method="POST">
+        <textarea name="comment-input">one, two, three</textarea>
```

Comment:
Allowing the viewer to submit arbitrary text as a comment here and later showing it in the comments section is vulnerable to an HTML [injection attack](https://en.wikipedia.org/wiki/Code_injection).

For example, if I type `<b>I can inject arbitrary HTML content</b>` into the comments page, and after retrieving comments later, that shows up as **bold** text. A malicious user can craft a comment that is a piece of HTML with a malicious payload included.

One way to address issues like this (and I'm not an expert by any means) would be to do some sanitization of the user-provided input, either when provided or when later rendered in the webpage, to make sure it doesn't "escape" its comment container.
jQuery's [text(text) function](https://api.jquery.com/text/#text-text) might be useful.